.gitignore

```diff
@@ -328,3 +328,6 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
+# VS Code
+.vscode/
```

Comment:
@svekukucska please, add a new line

Reply:
I will implement the requested changes and amend it to this PR.